cmc-ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/mapper/AddressMapperTest.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.hmcts.cmc.ccd.mapper;
+
+import org.junit.Test;
+import uk.gov.hmcts.cmc.domain.models.Address;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleAddress;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+
+public class AddressMapperTest {
+
+
+    Mapper<uk.gov.hmcts.cmc.ccd.domain.Address, Address> mapper = new AddressMapper();
+
+    @Test
+    public void shouldMapAddressToCCD() {
+        //given
+        Address address = SampleAddress.builder().build();
+
+        //when
+        uk.gov.hmcts.cmc.ccd.domain.Address ccdAddress = mapper.to(address);
+
+        //then
+        assertNotNull(ccdAddress);
```

Comment:
Please don't do assertions like this. If a method can return a null object it should return an `Optional` of the value (here and in all tests).

Reply:
removed